docs/dev-environments/besu/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+  #
+  node1.eea.org:
+    image: hyperledger/besu:1.2.4
+    command: --data-path=/var/lib/besu --genesis-file=/var/lib/besu/genesis.json --rpc-http-enabled --rpc-http-host=0.0.0.0 --rpc-ws-enabled --rpc-ws-host=0.0.0.0 --discovery-enabled=false --p2p-host=0.0.0.0 --rpc-http-api=ETH,NET,WEB3,DEBUG,MINER,ADMIN,TXPOOL,CLIQUE --rpc-ws-api=ETH,NET,WEB3,DEBUG,MINER,ADMIN,TXPOOL,CLIQUE --rpc-http-cors-origins='*' --host-whitelist='*' --min-gas-price=0
+    ports:
+      # RPC endpoint
+      - 22011:8545
+      # websocket endpoint (for subscribing to events)
+      - 22002:8546
+    volumes:
+      - './besu/node1:/var/lib/besu'
+    networks:
+      local_net:
+        ipv4_address: '172.13.0.2'
+
+  #
+  # ethsigner1 is the signing wallet associated to node1, and provides an RPC endpoint for web3 APIs
+  # This account is used by the application as the EEA Admin signing account
+  #
+  rpc.node1.eea.org:
+    image: pegasyseng/ethsigner
+    command: --chain-id=12345 --downstream-http-host=node1.eea.org --downstream-http-port=8545 --http-listen-host=0.0.0.0 --http-listen-port=8555 file-based-signer --password-file=/var/lib/ethsigner/password.txt --key-file=/var/lib/ethsigner/keystore/account.txt
+    ports:
+      - 22001:8555
+    volumes:
+      - './ethsigner/node1:/var/lib/ethsigner'
+    depends_on:
+      - node1.eea.org
+    networks:
+      local_net:
+        ipv4_address: '172.13.0.3'
+
+  #
+  # node2 is the second of the two-node ethereum private network based on Hyperledger Besu
+  #
+  node2.eea.org:
+    image: hyperledger/besu:1.2.4
+    command: --data-path=/var/lib/besu --genesis-file=/var/lib/besu/genesis.json --rpc-http-enabled --rpc-http-host=0.0.0.0 --rpc-ws-enabled --rpc-ws-host=0.0.0.0 --discovery-enabled=false --p2p-host=0.0.0.0 --rpc-http-api=ETH,NET,WEB3,DEBUG,MINER,ADMIN,TXPOOL,CLIQUE --rpc-ws-api=ETH,NET,WEB3,DEBUG,MINER,ADMIN,TXPOOL,CLIQUE --rpc-http-cors-origins='*' --host-whitelist='*' --min-gas-price=0
+    ports:
+      # RPC endpoint
+      - 23011:8545
+      # websocket endpoint (for subscribing to events)
+      - 23002:8546
+    volumes:
+      - './besu/node2:/var/lib/besu'
+    networks:
+      local_net:
+        ipv4_address: '172.13.0.4'
+
+  #
+  # ethsigner2 is a 2nd signing wallet associated to node1, and represents a signing account for organization 1
+  #
+  rpc.node2.eea.org:
+    image: pegasyseng/ethsigner
+    command: --chain-id=12345 --downstream-http-host=node2.eea.org --downstream-http-port=8545 --http-listen-host=0.0.0.0 --http-listen-port=8555 file-based-signer --password-file=/var/lib/ethsigner/password.txt --key-file=/var/lib/ethsigner/keystore/account.txt
+    ports:
+      - 23001:8555
+    volumes:
+      - './ethsigner/node2:/var/lib/ethsigner'
+    depends_on:
+      - node2.eea.org
+    networks:
+      local_net:
+        ipv4_address: '172.13.0.5'
+
+networks:
+  local_net:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+      - subnet: 172.13.0.0/16
```

Comment:
We would need to put the Avalon components in the same network. Is it?

Reply:
Yep to put them in the same network, in a separate docker-compose.yaml, the network can be referenced like the following:

```
networks:
  default:
    external:
      name: besu_local_net
```

then the besu endpoints can be used like:

```
RPC endpoint: http://rpc.node1.avalon.org:8555
WS endpont: ws://node1.avalon.org:8546
```